modules/post/multi/gather/grub_password.rb

```diff
@@ -0,0 +1,61 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Post
+  include Msf::Post::File
+  include Msf::Post::Unix
+
+  def initialize(info = {})
+    super(update_info(info,
+      'Name'         => 'Gather GRUB Password',
+      'Description'  => %q{
+        This module gathers GRUB passwords from GRUB bootloader config files.
+      },
+      'License'      => MSF_LICENSE,
+      'Author'       =>
+        [
+          'Garvit Dewan <d.garvit[at]gmail.com>' # @dgarvit
+        ],
+      'Platform'     => ['linux', 'osx', 'unix', 'solaris', 'bsd'],
+      'SessionTypes' => ['meterpreter', 'shell'],
+      'References'   => [
+        ['URL', 'https://help.ubuntu.com/community/Grub2/Passwords#Password_Encryption']
+      ]
+    ))
+  end
+
+  def run
```

Comment:
This module will produce no output, even with `verbose true`, if no GRUB config files are found.

This may be confusing for the user.

The lazy approach would be to simply `print_status 'Searching for GRUB config files...'` at the start.

Reply:
Fixed it!